node/service/src/lib.rs

```diff
@@ -704,7 +704,10 @@ pub fn new_full<RuntimeApi, Executor>(
 		// given delay.
 		let voting_rule = match grandpa_pause {
 			Some((block, delay)) => {
-				info!("GRANDPA scheduled voting pause set for block #{} with a duration of {} blocks.",
+				info!(
+					block_number = %block,
+					delay = %delay,
+					"GRANDPA scheduled voting pause set for block #{} with a duration of {} blocks.",
```

Comment:
sorry for being late to the party

there are a few places left where we print the data both in the message and via structured logging and it's not clear to me whether we need the former and what is the best practice here

Reply:
In cases like this where I retained both, it's because it wasn't immediately clear how to rewrite the message while preserving clarity.